### PR TITLE
kyma-cli: Add version 1.21.1

### PR DESCRIPTION
**Pull request by akelsch**
Homepage: https://kyma-project.io/
GitHub: https://github.com/kyma-project/cli

The CLI is available on Chocolatey as well.

### CODE REVIEW

File: bucket/kyma-cli.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            "hash": "46a261413ff74242599eca3b593694748e1b694bf58200e933759620ef1bfa0d"
+        },
+        "32bit": {
+            "url": "https://github.com/kyma-project/cli/releases/download/1.21.0/kyma_Windows_i386.zip",
+            "hash": "0ef097eff6e58890d66a539d77d37adfd81c7a81179dc3677f632f63dba2d2ac"
+        }
+    },
+    "bin": [
+        "kyma.exe"
+    ],
```
Comment (Ash258) [inline bucket/kyma-cli.json:18]:
```suggestion
    "bin":  "kyma.exe",
```

File: bucket/kyma-cli.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/kyma-project/cli/releases/download/$version/kyma_Windows_x86_64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/kyma-project/cli/releases/download/$version/kyma_Windows_i386.zip"
+            }
+        }
```
Comment (Ash258) [inline bucket/kyma-cli.json:28]: Add hash extraction from checksums.txt

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/kyma-cli.json (hunk 2 above)
Comment (akelsch) [reply bucket/kyma-cli.json:32]: Done in aa4176e. I have tested it locally using `kyma-cli@1.20.0` and it seems to work:

```
$ scoop install kyma-cli@1.20.0
WARN  Given version (1.20.0) does not match manifest (1.21.0)
WARN  Attempting to generate manifest for 'kyma-cli' (1.20.0)
Autoupdating kyma-cli
Searching hash for kyma_Windows_i386.zip in https://github.com/kyma-project/cli/releases/download/1.20.0/checksums.txt
Found: b2d7bae0f913f150b74891bbac29e631a03b330601b932c6a5684e5fb3519e33 using Extract Mode
Searching hash for kyma_Windows_x86_64.zip in https://github.com/kyma-project/cli/releases/download/1.20.0/checksums.txt
Found: 5f60dd184d7872ba5e2d78ab763dbc21641bfcfef470dd98dd1ce58114f3535e using Extract Mode
Writing updated kyma-cli manifest
Installing 'kyma-cli' (1.20.0) [64bit]
...
```

I am not familiar with the exact manifest syntax though so please check.

**Comment by Ash258** [comment]
Move this PR into [Main bucket](https://github.com/ScoopInstaller/Main)

**Pull request closed without merging**